Show bootstrap tooltip with full chat time.

markgeraty opened the pull request:
When hovering over a timestamp in the chat log, display the full timestamp in a bootstrap tooltip.

Closes #64 .

markgeraty committed:
Show bootstrap tooltip with full chat time.

When hovering over a timestamp in the chat log, display the full timestamp in a bootstrap tooltip.

markgeraty commented:
@swong225 can you test this out and merge when you are satisfied?  Thanks!

swong225 commented:
When you type in a new chat, if you hover over the timestamp, it displays as "Invalid date." You have to do a full refresh of the page in order for the proper date to be displayed.

markgeraty committed:
Fixing broken timestamp hover.

Showed invalid date when the user hovered over a message that they had just typed.  We were specifying the date as ISO8601, which was correct for the dates coming from the server, but not for the date that had just been created on the client.

Moment does not appear to need us to specify that, though, so just removing it fixes the issue.
